README.md

```diff
@@ -58,6 +58,7 @@ To further configure your Datadog construct, use the following custom parameters
 | `addLayers`          | Whether to add the Lambda Layers or expect the user to bring their own. Defaults to true. When true, the Lambda Library version variables are also required. When false, you must include the Datadog Lambda library in your functions' deployment packages.         |
 | `pythonLayerVersion` | Version of the Python Lambda layer to install, such as 21. Required if you are deploying at least one Lambda function written in Python and `addLayers` is true. Find the latest version number from [https://github.com/DataDog/datadog-lambda-python/releases][5]. |
 | `nodeLayerVersion`   | Version of the Node.js Lambda layer to install, such as 29. Required if you are deploying at least one Lambda function written in Node.js and `addLayers` is true. Find the latest version number from [https://github.com/DataDog/datadog-lambda-js/releases][6].   |
+| `extensionLayerVersion`   | Version of the Datadog Lambda Extension layer to install, such as 5. The Datadog Lambda Extension layer is in public preview. Learn more about the lambda extension here [https://docs.datadoghq.com/serverless/datadog_lambda_library/extension][12].   |
```

Comment:
Nit: I think you want to make it so that the word `here` is the link

Reply:
Modified, will be present in next commit